styles/main.css

```diff
@@ -0,0 +1,199 @@
+body {
+  font-family: sans-serif;
+  width: 98%;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 50px;
+}
+
+img {
+  margin: 10px;
+}
+
+body section {
+  border-bottom: solid 1px grey;
+  padding-top: 50px;
+  padding-bottom: 50px;
+}
+
+#why {
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+
+#page-intro h1 {
+  text-align: center;
+  font-size: 90px;
+  margin-top: 150px;
+  font-family: 'Bungee Outline', sans-serif;
+  color: purple;
+}
+
+#page-intro .button {
+  margin-bottom: 100px;
+}
+
+h2 {
+  text-align: center;
+  font-size: 28px;
+  margin-top: 40px;
+  margin-bottom: 20px;
+}
+
+ul {
+  list-style-type: none;
+  border: 0px;
+  padding: 0px;
+}
+
+sub {
+  font-size: 30px;
+  text-transform: capitalize;
+  font-weight: bold;
+  display: block;
+  text-align: center;
+  margin-top: 55px;
+  margin-bottom: 80px;
+}
+
+.button {
+  border: solid 4px;
+  border-radius: 12px;
+  border-color: lightgrey;
+  display: block;
+  width: 270px;
+  padding: 22px 20px 22px 20px;
+  font-size: 28px;
+  text-align: center;
+  text-decoration: none;
+  color: inherit;
+  margin: auto;
+}
+
+.button:hover {
+  background-color: grey;
+  color: white;
+}
+
+#intros {
+  border-bottom: none;
+}
+
+#intros img {
+  border-radius: 100%;
```

Comment:
You made the images black and white and gave them a border. Keeping consistent with the mock up!! 🌟